plexpy/libraries.py

```diff
@@ -768,7 +768,6 @@ def get_watch_time_stats(self, section_id=None, grouping=None):
 
         monitor_db = database.MonitorDatabase()
 
-        time_queries = [1, 7, 30, 0]
```

Comment:
The default `time_queries` should be added here otherwise a call to `get_watch_time_stats` without any `time_queries` will fail below.

```
if time_queries is None:
    time_querires = [1, 7, 30, 0]
```